en-US/focus-ios.xliff

```diff
@@ -951,6 +975,11 @@
         <target>Tap the trash anytime to remove all traces of your current session.</target>
         <note>This is the body text that is displayed for the Trash icon tooltip</note>
       </trans-unit>
+      <trans-unit id="TooltipBodyText.TrashIcon.V2">
+        <source>Tap here to trash it all—history, cookies, everything—and start fresh on a new tab.</source>
```

Comment:
Dashes should have a space before and after, per copy guidelines
https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FIREFOX&title=Punctuation#Punctuation-Hyphensanddashes

Reply:
This should be fixed by #3446